packages/queries/src/query.ts

```diff
@@ -100,10 +100,41 @@ export class QueryBuilder<
         );
     }
 
+    /**
+     * Returns all rows that match the query filters.
```

Comment:
We could also use the same "Builds a query that" language from the other two here, or change the other two